tests/unit/amo/reducers/test_addonsByAuthors.js

```diff
@@ -0,0 +1,170 @@
+import { ADDON_TYPE_THEME } from 'core/constants';
+import reducer, {
+  OTHER_ADDONS_BY_AUTHORS_PAGE_SIZE,
+  fetchOtherAddonsByAuthors,
+  initialState,
+  loadOtherAddonsByAuthors,
+} from 'amo/reducers/addonsByAuthors';
+import { fakeAddon } from 'tests/unit/amo/helpers';
+
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes properly', () => {
+      const state = reducer(undefined, {});
+      expect(state).toEqual(initialState);
+    });
+
+    it('ignore unrelated actions', () => {
+      // Load some initial state to be sure that an unrelated action does not
+      // change it.
+      const state = reducer(undefined, loadOtherAddonsByAuthors({
+        slug: fakeAddon.slug,
+        addons: [fakeAddon],
+      }));
+      const newState = reducer(state, { type: 'UNRELATED' });
+      expect(newState).toEqual(state);
+    });
+
+    describe('loadOtherAddonsByAuthors()', () => {
```

Comment:
I think all of these can be merged into the other `loadOtherAddonsByAuthors` describe block